reaktive/src/commonMain/kotlin/com/badoo/reaktive/observable/Scan.kt

```diff
@@ -0,0 +1,41 @@
+package com.badoo.reaktive.observable
+
+import com.badoo.reaktive.completable.CompletableCallbacks
+import com.badoo.reaktive.disposable.Disposable
+import com.badoo.reaktive.utils.Uninitialized
+import com.badoo.reaktive.utils.lock.newLock
+import com.badoo.reaktive.utils.lock.synchronized
+
+@Suppress("UNCHECKED_CAST")
```

Comment:
Please apply suppress annotations to statements, it will help preventing unsafe casts in other places.